tests/cnf/core/network/sriov/tests/qinq.go

```diff
@@ -885,3 +1065,133 @@ func runQinQDpdkTestCases(nodeName, serverName, clientName, sriovNetworkName, na
 	By("Validate that the TCP traffic is double tagged")
 	readAndValidateTCPDump(tcpDumpPod, []string{"bash", "-c", "tail -20 /tmp/tcpdump"}, outPutSubString)
 }
+
+func defineNetworkBondAnnotation(sVlan, cvlan string, ipAddresses []string) []*multus.NetworkSelectionElement {
```

Comment:
What's the reason to have this one line func?

Reply:
removed